docker/nginx.conf

```diff
@@ -0,0 +1,48 @@
+worker_processes      1;
+
+events {
+  worker_connections  1024;
+}
+
+http {
+  include             mime.types;
+  default_type        application/octet-stream;
+
+  sendfile on;
+
+  keepalive_timeout   65;
+
+  gzip on;
+  gzip_static on;
+  gzip_disable "msie6";
+
+  gzip_vary on;
+  gzip_types text/plain text/css application/javascript;
+
+  map $request_method $access_control_max_age {
+    OPTIONS 1728000; # 20 days
+  }
+  add_header X-Frame-Options deny;
```

Comment:
I'm similarly unsure about whether this is still needed. It's carry over from the original docker-run.sh in this repository.

Reply:
I don't see why that would need to be here. (It prevents the tool from being included in an iframe.) But any change to that functionality would be beyond the scope of this.